src/utils/exportUtils/Exporter.ts

```diff
@@ -75,9 +75,18 @@ export default abstract class Exporter {
         this.files.push(file);
     }
 
+    protected santizeFileName(filename: string): string {
+        filename = filename.normalize('NFD').replace(/[\u0300-\u036f]/g, "");
+        filename = filename.replace(/[^a-z0-9áéíóúñü.,_-]/gim, "");
+        filename = filename.replace(/[ ]/gim, "-");
```

Comment:
I think this might need to be swapped with the line above, otherwise that line will replace spaces with `""`